src/main/java/knu/kproject/controller/BoarderController.java

```diff
@@ -34,11 +34,8 @@ public ResponseEntity<Api_Response<Object>> createBoard(@AuthenticationPrincipal
         Map<String, UUID> boarderMap = new HashMap<>();
         boarderMap.put("BoarderId", boarderId);
 
-        return ApiResponseUtil.createResponse(
-                SuccessCode.INSERT_SUCCESS.getStatus(),
-                SuccessCode.INSERT_SUCCESS.getMessage(),
-                boarderMap
-        );
+        return ApiResponseUtil.createSuccessResponse(
+                SuccessCode.INSERT_SUCCESS.getMessage());
```

Comment:
```suggestion
        return ApiResponseUtil.createResponse(
                SuccessCode.INSERT_SUCCESS.getStatus(),
                SuccessCode.INSERT_SUCCESS.getMessage(),
                boarderMap
        );
```
이렇게 사용한 이유가 보드 생성 후 생성한 보드의 id를 반환해줄려고 일부러 이렇게 만들었던거야.

Reply:
그럼 이건 둘 다 원래대로 돌려서 바로 pr 날리겠습니다